sw/host/tests/usbdev/usbdev_stream/stream_test.h

```diff
@@ -0,0 +1,79 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+#ifndef OPENTITAN_SW_HOST_TESTS_USBDEV_USBDEV_STREAM_STREAM_TEST_H_
+#define OPENTITAN_SW_HOST_TESTS_USBDEV_USBDEV_STREAM_STREAM_TEST_H_
+#include <cstdint>
+
+// Maximum number of concurrent streams, leaving at least one endpoint for
+// the Default Control Pipe.
+#ifdef USBDEV_NUM_ENDPOINTS
+#define STREAMS_MAX (USBDEV_NUM_ENDPOINTS - 1U)
+#else
+#define STREAMS_MAX 11U
+#endif
+
+/**
+ * Simple container for test configuration.
+ */
+struct TestConfig {
+  /**
+   * Initialize test settings.
```

Comment:
Maybe document arguments?

Reply:
Added a comment; should suffice since it's only a simple aggregation of the various configuration settings.